layer/compile_time_layer.cc

```diff
@@ -164,11 +171,11 @@ SPL_COMPILE_TIME_LAYER_FUNC(VkResult, CreateComputePipelines,
   assert(create_info_count > 0 &&
          "Specification says create_info_count must be > 0.");
 
-  absl::Time start = absl::Now();
+  auto start = Now();
```

Comment:
We should spell out the exact type because it's not obvious by just looking at the RHS. Also everywhere else,